packages/flutter_tools/lib/src/persistent_tool_state.dart

```diff
@@ -78,6 +81,7 @@ class _DefaultPersistentToolState implements PersistentToolState {
     Channel.beta: 'last-active-beta-version',
     Channel.stable: 'last-active-stable-version'
   };
+  static const String _kAzureBot = 'azure-bot';
```

Comment:
maybe 'is-bot' since this is no longer azure specific.

Reply:
Done